src/checks/incorrect_delimiter.rs

```diff
@@ -19,7 +21,7 @@ impl Check for IncorrectDelimiterChecker {
         if key.trim().chars().any(|c| !c.is_alphanumeric() && c != '_') {
             return Some(Warning::new(
                 line.clone(),
-                self.template.replace("{}", &key),
+                format!("{}: {}", self.name, self.template.replace("{}", &key)),
```

Comment:
Could you please move this line to the `message` function? For example:
https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/unordered_keys.rs#L11

Reply:
Yeah, I've added it.